REDCapUITweaker.php

```diff
@@ -7,24 +7,63 @@ class REDCapUITweaker extends \ExternalModules\AbstractExternalModule
 {
 	const SUBMIT_TYPES = [ 'record', 'continue', 'nextinstance',
 	                       'nextform', 'nextrecord', 'exitrecord' ];
+	const SUBMIT_DEFINE = 'record,nextinstance,nextform,continue';
 
 	private $customAlerts;
 
 
+
+
+
 	// Initialise module when enabled.
 	// Set default system settings and enable in all projects.
 
 	function redcap_module_system_enable()
 	{
-		$this->setSystemSetting( 'field-types-order', '1,2,7,5,6,11,3|4,8,10,9' );
-		$this->setSystemSetting( 'field-default-required', '1' );
-		$this->setSystemSetting( 'submit-option-tweak', '0' );
+		if ( $this->getSystemSetting( 'field-types-order' ) == '' &&
+		     $this->getSystemSetting( 'field-default-required' ) == '' &&
+		     $this->getSystemSetting( 'submit-option-tweak' ) == '' )
+		{
+			$this->setSystemSetting( 'field-types-order', '1,2,7,5,6,11,3|4,8,10,9' );
```

Comment:
Field types/numbers are listed in the readme file, but it would be nice to have it in a comment here so one doesn't have to do a lookup elsewhere.

Reply:
Comments now added.